### ISSUE TITLE
Very tiny bug on pls triv

### ISSUE THREAD

**Issue by Themis7777** [DankMemer/dank-memer-suggestions-and-bugs#1083]
On one question, the correct answer is not italicized.

I expected all the questions to be the same format so the correct answer wouldn't be obvious

![image](https://user-images.githubusercontent.com/68445543/87828355-c49d4e00-c84a-11ea-9b9b-83c453a1e26f.png)

User ID: 408973164002017281

This is a small bug but still a problem nonetheless

**Comment by Wise2006** [comment]
Yeah. I was thinking about reporting this but since it was a tiny bug, I didn't.
